linux/sound.cpp

```diff
@@ -87,7 +77,7 @@ void CSound::OpenJack ( const bool bNoAutoJackConnect, const char* jackClientNam
 
     if ( ( input_port_left == nullptr ) || ( input_port_right == nullptr ) || ( output_port_left == nullptr ) || ( output_port_right == nullptr ) )
     {
-        throw CGenErr ( tr ( "The Jack port registering failed." ) );
+        throw CGenErr ( tr ( "The JACK port registering failed." ) );
```

Comment:
I'm not too familiar with this, but is there a reason for the failure?

Reply:
Not sure code wise.

Reply:
Hm. That's tricky. 
At the moment, it's a dead-end error. Is there a general solution for this? Rebooting?

Reply:
It's "stop Jamulus, stop JACK, start JACK and see if another client can connect at 48KHz (e.g. Audacity)" or something like that.  But you're into diagnosing a JACK failure, really, rather than a Jamulus failure.